normandy/control/static/control/js/components/ApprovalRequestCommentForm.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { reduxForm, getValues } from 'redux-form'
+
+import ControlActions from '../actions/ControlActions.js'
+
+
+export class ApprovalRequestCommentForm extends React.Component {
+  submitForm() {
+    const { dispatch, formState, approvalRequest } = this.props;
+    let values = getValues(formState.approvalRequestComment);
+    dispatch(ControlActions.makeApiRequest('createApprovalRequestComment', {approvalRequestId: approvalRequest.id, ...values}));
+  }
+
+  render() {
+    const { fields: { text }, approvalRequest, handleSubmit } = this.props;
+
+    return (
+      <form id="comment-form" onSubmit={handleSubmit(::this.submitForm)}>
+        <div className="fluid-6 row">
+          <textarea placeholder="Add comment&hellip;" field={text} {...text} />
```

Comment:
Having HTML entities in this code feels weird, and the React docs [imply we might run into double escaping issues](https://facebook.github.io/react/docs/jsx-gotchas.html). Can we just put the Unicode character here? `…`? Or is that gross?